test/stage2/compare_output.zig

```diff
@@ -17,9 +17,9 @@ pub fn addCases(ctx: *TestContext) !void {
     }
 
     {
-        var case = ctx.addExe("hello world with updates", linux_x64);
+        var case = try ctx.addExe("hello world with updates", linux_x64, .Zig);
```

Comment:
You shouldn't have to pass `.Zig` here, that should be the default, and ZIR tests are the exceptional cases.

Reply:
So, an anonymous struct with default args?

Reply:
.exe / .obj / etc, as discussed in IRC; fixing shortly.